sdk/tools/openrewrite-sample-compiler-ci.yml

```diff
@@ -0,0 +1,73 @@
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - /sdk/tools/
+
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages: 
+      - stage: CompileSamples
+        variables: 
+          - template: /eng/pipelines/templates/variables/globals.yml
+          - template: /eng/pipelines/templates/variables/image.yml
+
+        jobs:
+          - job: OpenRewriteCompilerPlugin
+            pool: 
+              name: $(LINUXPOOL)
+              image: $(LINUXVMIMAGE)
+              os: linux
+            displayName: Build and Run OpenRewrite Compiler Plugin
+            timeoutInMinutes: 60
+            steps:
+              - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+                parameters:
+                  Paths:
+                    - /sdk/tools/azure-openrewrite-compiler-maven-plugin/
+                    - /sdk/tools/azure-openrewrite/
+                    - /sdk/core-v2/
```

Comment:
Should we include `sdk/identity-v2` as well?

Reply:
added